### PR TITLE
fix: pile chi calculation

### PR DESCRIPTION
**Pull request by manuelpolzhofer**

### CODE REVIEW

File: src/pile.sol [hunk 1]
```diff
@@ -74,7 +74,9 @@ contract Pile is DSNote {
     }
     
     function file(uint fee, uint speed_) public auth note {
-        fees[fee].speed = speed_; 
+        fees[fee].speed = speed_;
+        fees[fee].chi = ONE;
+        drip(fee);
```
Comment (lucasvo) [inline src/pile.sol]: this drip should be called before changing the speed IMO.

File: src/test/pile.t.sol [hunk 2]
```diff
@@ -20,13 +20,20 @@ import "ds-test/test.sol";
...
 contract PileTest is DSTest {
     Pile pile;
     TokenMock tkn;
 
-    function setUp() public {
+    Hevm hevm;
 
+    function setUp() public {
+        hevm = Hevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+        hevm.warp(0);
```
Comment (lucasvo) [inline src/test/pile.t.sol]: Can you test with something other than warp(0)? Check this out: https://github.com/makerdao/dss/pull/45/files#diff-ba2ea047fd34da8c18c72fcc967b6e4bL115 (setting it to 0 was the problem why we didn't find the rho issue)

File: src/test/pile.t.sol [hunk 3]
```diff
@@ -58,7 +64,7 @@ contract PileTest is DSTest {
 
         pile.withdraw(loan,wad,address(this));
 
-        assertEq(totalBalance-wad, pile.Balance());
+         assertEq(totalBalance-wad, pile.Balance());
```
Comment (lucasvo) [inline src/test/pile.t.sol]: spacing

**Comment by manuelpolzhofer** [comment]
removed `hvem.warp(0)` from tests

File: src/pile.sol (hunk 1 above)
Comment (manuelpolzhofer) [reply src/pile.sol]: Both ways work.

**Pull request merged**